internal/schema/v1/location.go

```diff
@@ -0,0 +1,34 @@
+package schema
+
+import "github.com/Decentr-net/cerberus/internal/schema/types"
+
+// Location is user's geolocation.
+type Location struct {
+	Timestamp
+
+	Latitude  float64 `json:"latitude"`
```

Comment:
If there is a specific web site asked for location e.g. maps.google.com it would be nice to include this information into the location PDV (optional param)

Reply:
done